src/test/java/com/google/cloud/spanner/jdbc/it/ITJdbcPgDatabaseMetaDataTest.java

```diff
@@ -57,12 +54,12 @@ public class ITJdbcPgDatabaseMetaDataTest extends ITAbstractJdbcTest {
   private static final String TABLE_WITH_ALL_COLS = "TableWithAllColumnTypes";
   private static final String TABLE_WITH_REF = "TableWithRef";
 
-  @BeforeClass
-  public static void skipOnEmulator() {
-    assumeFalse(
-        "PostgreSQL dialect is not yet supported on the emulator",
-        EmulatorSpannerHelper.isUsingEmulator());
-  }
+  //  @BeforeClass
```

Comment:
Should we delete the code rather than commenting?

Reply:
Thanks for spotting, removed.